opensearch/chart/values.yaml

```diff
@@ -406,6 +406,22 @@ cluster:
             memory: "1Gi"
             cpu: "500m"
 
+    # -- Configuration for the client service used to expose client nodes externally.
+    client:
+      service:
+        # -- Enable or disable the external client service.
+        enabled: true
+        # -- The type of Kubernetes service to create. Default is LoadBalancer for external access.
+        type: LoadBalancer
```

Comment:
Why do we want to use `LoadBalancer` by default? By installing this you would potentially start an external service to anyone without knowing it, no?